lib/src/screens/dashboard/widgets/sync_indicator_icon.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:cake_wallet/palette.dart';
+
+class SyncIndicatorIcon extends StatelessWidget {
+   SyncIndicatorIcon({this.isSynced});
+
+   final bool isSynced;
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 4,
+      width: 4,
+      decoration: BoxDecoration(
+          shape: BoxShape.circle,
+          color: isSynced
+              ? PaletteDark.brightGreen
+              : Theme.of(context).textTheme.caption.color
+      ),
+    );
+  }
```

Comment:
Add one more tab. Add 1 empty line above, between `@override` and `final bool isSynced`